packages/user-facing-text/src/validations.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Validations text
+ */
+
+import { Validations } from "../api";
+
+// ESLint will remove the ":string" causing an error in "implementation-matches-public-api.ts"
+// eslint-disable-next-line @typescript-eslint/no-inferrable-types
+export const DIAGNOSTIC_SOURCE: string = "UI5 Language Assistant";
+
+export const validations: Validations = {
+  UNKNOWN_CLASS_IN_NS: {
+    msg: `The "{0}" class does not exist in the "{1}" namespace`,
+    code: 1001,
+  },
+  UNKNOWN_CLASS_WITHOUT_NS: {
+    msg: `The "{0}" class does not exist, please specify a namespace`,
+    code: 1002,
+  },
+  UNKNOWN_AGGREGATION_IN_CLASS: {
+    msg: `The "{0}" aggregation does not exist in the "{1}" class`,
+    code: 1003,
+  },
+  UNKNOWN_AGGREGATION_IN_CLASS_DIFF_NAMESPACE: {
+    msg: `The "{0}" aggregation must have the same namespace as the "{1}" class`,
+    code: 1004,
+  },
+  UNKNOWN_TAG_NAME_IN_CLASS: {
+    msg: `The "{0}" name is neither a class name nor an aggregation of the "{1}" class`,
+    code: 1005,
+  },
+  UNKNOWN_TAG_NAME_IN_NS_UNDER_CLASS: {
+    msg: `The "{0}" name is neither a class name in the "{1}" namespace nor an aggregation of the "{2}" class`,
+    code: 1006,
+  },
+  UNKNOWN_TAG_NAME_IN_NS: {
+    msg: `The "{0}" name is neither a class name in the "{1}" namespace nor an aggregation of its parent tag`,
+    code: 1007,
+  },
+  UNKNOWN_TAG_NAME_NO_NS: {
+    msg: `The "{0}" name is neither a class name nor an aggregation of its parent tag, please specify a namespace`,
```

Comment:
The "{0}" name is neither a class name nor an aggregation of its parent tag. Specify a namespace.
[Can we say "Enter a namespace." or "Choose a namespace." instead of "Specify"?
Enter if typed in and choose/select is chosen from a list.]